### PR TITLE
Fix akima install

### PR DESCRIPTION
**Pull request by jsolbrig**
# Reviewer Checklist

PR author: Please ensure you meet all of the below requirements, and check boxes appropriately.

Reviewers: Please confirm all required testing/documentation has been completed prior to approving.

**Remove lines that are not applicable, explain if you select "NO REQUIRED"**

* [ ] Required ***existing tests*** pass (ie full_test.sh, others as appropriate)
* [ ] NO REQUIRED ***existing tests*** (explain why not required)
* [ ] Required ***unit tests*** added and pass for new/modified functionality
* [ ] NO REQUIRED ***unit tests*** (explain why not required)
* [ ] Required ***integration tests*** added and pass for new/modified functionality
* [ ] NO REQUIRED ***integration tests*** (explain why not required)
* [ ] Required ***documentation*** added for new/modified functionality
* [ ] NO REQUIRED ***documentation*** (explain why not required)
* [ ] Required ***release notes*** added for new/modified functionality
* [ ] NO REQUIRED ***release notes*** (explain why not required)
* [ ] Required ***updates to other repos*** complete
* [ ] NO REQUIRED ***updates to other repos*** (explain why not required)

https://github.com/NRLMMD-GEOIPS/.github/blob/main/.github/review-template.md

# Related Issues
fixes NRLMMD-GEOIPS/geoips#NNN

# Testing Instructions

# Summary

# Output

### CODE REVIEW

**Review by mindyls — DISMISSED** [review]
Looks good to me - I uninstalled akima86 and reinstalled recenter_tc and everything works:

```
Downloading akima86-0.1.1-cp310-cp310-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.1 MB)
   ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 1.1/1.1 MB 12.9 MB/s eta 0:00:00
```

```

Return values:
  Python install: 
  sphinx html:    0
  sphinx latex:   None
  latex pdf make: None
  warnings html:  
  warnings latex: 
  warnings make:  

Final retval: 0
```

```
23_213457  recenter_tc.py:339     INFO: USING recenter variable name akima
23_213457    log_setup.py:50      INFO: ********************************************************************************
23_213457    log_setup.py:55      INFO: ** USING recenter variable name akima area def                                **
23_213457    log_setup.py:55      INFO: ** Area ID: tc2021wp20mindulle Description: tc2021wp20mindulle                **
23_213457    log_setup.py:55      INFO: ** interpolated_time 2021-09-26 21:06:00 Projection ID:                       **
23_213457    log_setup.py:55      INFO: ** eqc_tc2021wp20mindulle Projection: {'R': '6371228', 'lat_0': '19.5',       **
23_213457    log_setup.py:55      INFO: ** 'lat_ts': '0', 'lon_0': '136.7', 'no_defs': 'None', 'proj': 'eqc', 'type': **
23_213457    log_setup.py:55      INFO: ** 'crs', 'units': 'm', 'x_0': '0', 'y_0': '0'} Number of columns: 1400       **
23_213457    log_setup.py:55      INFO: ** Number of rows: 1400 Area extent: (-700000.0, -700000.0, 700000.0,         **
23_213457    log_setup.py:55      INFO: ** 700000.0)                                                                  **
23_213457    log_setup.py:55      INFO: ** source_sector_file:                                                        **
23_213457    log_setup.py:55      INFO: ** $GEOIPS_PACKAGES_DIR/geoips/tests/sectors/tc_bdecks/bwp202021.dat          **
23_213457    log_setup.py:55      INFO: ** deck_line: WP, 20, 2021092700,   , BEST,   0, 196N, 1367E, 100,  952, TY,  **
23_213457    log_setup.py:55      INFO: ** 34, NEQ,  215,  200,  140,  175, 1005,  300,  15,   0,  20,   W,   0,    , **
23_213457    log_setup.py:55      INFO: ** 0,   0,   MINDULLE, D,                                                     **
23_213457    log_setup.py:55      INFO: ** storm_basin: WP                                                            **
23_213457    log_setup.py:55      INFO: ** storm_num: 20                                                              **
23_213457    log_setup.py:55      INFO: ** synoptic_time: 2021-09-27 00:00:00                                         **
23_213457    log_setup.py:55      INFO: ** storm_start_datetime: 2021-09-20 12:00:00                                  **
23_213457    log_setup.py:55      INFO: ** aid_type: BEST                                                             **
23_213457    log_setup.py:55      INFO: ** clat: 19.5                                                                 **
23_213457    log_setup.py:55      INFO: ** clon: 136.7                                                                **
23_213457    log_setup.py:55      INFO: ** vmax: 103.52                                                               **
23_213457    log_setup.py:55      INFO: ** pressure: 949.82                                                           **
23_213457    log_setup.py:55      INFO: ** storm_name: MINDULLE                                                       **
23_213457    log_setup.py:55      INFO: ** final_storm_name: MINDULLE                                                 **
23_213457    log_setup.py:55      INFO: ** storm_year: 2021                                                           **
23_213457    log_setup.py:55      INFO: ** invest_number: 99                                                          **
23_213457    log_setup.py:55      INFO: ** source_filename:                                                           **
23_213457    log_setup.py:55      INFO: ** $GEOIPS_PACKAGES_DIR/geoips/tests/sectors/tc_bdecks/bwp202021.dat          **
23_213457    log_setup.py:55      INFO: ** parser_name: bdeck_parser                                                  **
23_213457    log_setup.py:55      INFO: ** interpolated_time: 2021-09-26 21:06:00                                     **
23_213457    log_setup.py:55      INFO: ** recenter_type: akima                                                       **
23_213457    log_setup.py:55      INFO: ** adjustment_id: akima                                                       **
23_213457    log_setup.py:56      INFO: ********************************************************************************
23_213457    log_setup.py:57      INFO:

23_213457  recenter_tc.py:347     INFO:

out_fnames
[]
23_213457    log_setup.py:50   INTERACTIVE: ********************************
23_213457    log_setup.py:55   INTERACTIVE: ** Done recentering TC sector **
23_213457    log_setup.py:56   INTERACTIVE: ********************************
```

```
23_213509 single_source.py:2258 INTERACTIVE:     SINGLESOURCESUCCESS ${GEOIPS_OUTDIRS}/preprocessed/tcwww/tc2021/WP/WP202021/png_clean/windspeed/smap/20210926_085600_WP202021_smap-spd_smap_windspeed_139kts_54p35_res1p0-akima-clean.png
23_213509 single_source.py:2258 INTERACTIVE:     SINGLESOURCESUCCESS ${GEOIPS_OUTDIRS}/preprocessed/tcwww/tc2021/WP/WP202021/metadata/sector_information/20210926/20210926_085600_WP202021_smap-spd_smap_windspeed_139kts_54p35_res1p0-akima-clean.png.yaml
23_213509 single_source.py:2258 INTERACTIVE:     SINGLESOURCESUCCESS ${GEOIPS_OUTDIRS}/preprocessed/tcwww/tc2021/WP/WP202021/png_clean/windspeed/smap/20210926_210400_WP202021_smap-spd_smap_windspeed_104kts_74p70_res1p0-akima-clean.png
23_213509 single_source.py:2258 INTERACTIVE:     SINGLESOURCESUCCESS ${GEOIPS_OUTDIRS}/preprocessed/tcwww/tc2021/WP/WP202021/metadata/sector_information/20210926/20210926_210400_WP202021_smap-spd_smap_windspeed_104kts_74p70_res1p0-akima-clean.png.yaml
23_213533 single_source.py:2269 INTERACTIVE: READER_NAME: smap_remss_winds_netcdf
23_213533 single_source.py:2270 INTERACTIVE: PRODUCT_NAME: windspeed
23_213533 single_source.py:2271 INTERACTIVE: NUM_PRODUCTS: 4
23_213533 single_source.py:2272 INTERACTIVE: NUM_DELETED_PRODUCTS: 0
23_213533 geoips_utils.py:177     INFO: Total Time 0:00:06.596840: 2 Num jobs: single_source procflow
23_213533 geoips_utils.py:187     INFO:     SUCCESS Process Time single_source procflow: tc2021wp20mindulle  : 0:00:01.756316
23_213533       memusg.py:267     INFO: Stopping resource tracking.
23_213533 run_procflow.py:68   INTERACTIVE: Completed geoips PROCFLOW single_source processing, done!
23_213533 run_procflow.py:72      INFO: Starting time: 2024-07-23 21:34:50.754654
23_213533 run_procflow.py:73      INFO: Ending time: 2024-07-23 21:35:33.656821
23_213533 run_procflow.py:74   INTERACTIVE: Total time: 0:00:42.902206
23_213533 run_procflow.py:94   INTERACTIVE: Return value: 0
```

File: docs/source/releases/index.rst [hunk 1]
```diff
@@ -18,6 +18,14 @@
...
 *************
 
+Latest (cutting-edge from GitHub)
+------
+
+.. toctree::
+   :maxdepth: 1
+
+   latest
+
```
Comment (mindyls) [inline docs/source/releases/index.rst:28]: This actually breaks it - now build_docs.sh automatically u pdates this file.  You shouldn't update it directly.

Comment (jsolbrig) [reply docs/source/releases/index.rst]: This is fixed.

**Review by biosafetylvl5 — APPROVED** [review]
LGTM ! Thanks for changing this @jsolbrig

**Review by mindyls — APPROVED** [review]

**Comment by mindyls** [comment]
Merging!  Need to remove a blank line fro index.rst, so rst diff check fails correctly.

image

**Pull request merged**